updater/README.md

```diff
@@ -72,7 +72,7 @@ The updater may be run in two modes:
 
 The updater’s log is accessible as follows:
 ```sh
-journalctl -f --user-unit hubble-enterprise.service
+journalctl -fu hubble-enterprise.service
```

Comment:
this change is unrelated, right?

Reply:
No, it isn’t. `-u` stands for `--unit` and refers to system units, while `--user-unit` is the equivalent for user units and doesn’t have a corresponding shorthand option.